virttest/qemu_vm.py

```diff
@@ -1205,6 +1205,9 @@ def add_boot(devices, boot_order, boot_once, boot_menu, boot_strict):
                 cmd += " %s" % ",".join(options)
             else:
                 cmd = ""
+            if params.get("seabios_reboottimeout"):
+                reboot_timeout = " reboot-timeout = %s" % params.get('seabios_reboottimeout', 60)
```

Comment:
qemu command won't permit space around `=` with parameter, so `reboot-timeout=%s`

Reply:
@balamuruhans Thanks, have updated it.